docs/index.rst

```diff
@@ -12,7 +12,7 @@ eProsima Integration-Services Documentation
 
 *eProsima Integration-Service* is a tool based on `SOSS <https://github.com/eProsima/soss_v2>`__ and its
 **System-Handle** `SOSS-DDS <https://github.com/eProsima/SOSS-DDS>`__ that allows intercommunicating any
-:code:`DDS`-based system with any other protocol, including other :code:`DDS` systems, integrating them into a larger,
+:code:`DDS`-based system with any other protocol, including other :code:`DDS` protocols, integrating them into a larger,
```

Comment:
`systems` was better. `DDS` is a middleware that uses the `RTPS` protocol. There are `DDS` systems with different configurations, but there are not different `DDS` protocols.

Reply:
ok.